docs/tutorials/weaviate.md

```diff
@@ -0,0 +1,227 @@
+# Weaviate with local autoscaling embedding and generative models
+
+Weaviate is a vector search engine that can integrate seamlessly with KubeAI's embedding and generative models. This tutorial demonstrates how to deploy both KubeAI and Weaviate in a Kubernetes cluster, using KubeAI as the OpenAI endpoint for Weaviate.
```

Comment:
Maybe add a quick blurb about why you might want to do this: all of your data stays within your cluster.